syntax = "proto3";

option java_multiple_files = true;
option java_package = "org.fresno.adapter.grpc";
option java_outer_classname = "FresnoDailyNewsGetVerbsStartWithService";

service FresnoDailyNewsGetVerbsStartWith {

  rpc GetVerbsStartWith (GetVerbsStartWithRequest) returns (GetVerbsStartWithResponse) {}
}

message GetVerbsStartWithRequest {
  string message = 1;
}

message GetVerbsStartWithResponse {
  string reply = 1;
}

